arches/app/templates/views/components/cards/default.htm

```diff
@@ -1,7 +1,7 @@
 {% load i18n %}
 <!-- ko foreach: { data: [$data], as: 'self' } -->
 
-<!-- ko if: state === 'editor-tree' && self.card.model.visible() -->
```

Comment:
This causes cards cards that should not be visible to appear in the tree.

Reply:
resolved!